### PR TITLE
Updated user management docs — Renaming roles

### PR DESCRIPTION
**Pull request by validbeck**
## Internal Notes for Reviewers

> For sc-5420, I updated the guides in "Managing users" to reflect new understanding of groups, roles, and permissions.

### Managing users

The summary blurbs for guides under this section are now... informational! Instead of junk placeholder text that vaguely imply you should use these features for unclear purposes.

| Old | New |
|---|---|
| Screenshot 2024-08-28 at 11 15 44 AM | Screenshot 2024-08-28 at 11 10 04 AM |

### Manage roles
Updated this to reflect the new ability to rename roles. 

| Old | New |
|---|---|
|Screenshot 2024-08-28 at 11 17 14 AM | Screenshot 2024-08-28 at 11 10 23 AM|
| ... | Screenshot 2024-08-28 at 11 10 30 AM |

### CODE REVIEW

**Comment by validbeck** [comment]
@gtagle Can I lean on you to confirm accuracy of the new blurbs? :)

File: site/guide/configuration/manage-permissions.qmd [hunk 1]
```diff
@@ -17,7 +17,7 @@ Use permissions to streamline and secure access control within your organization
 
 ## Steps
 
-::: {.callout-important title="Be careful when modifying permissions to ensure they align with the role's intended responsibilities and access requirements."}
+::: {.callout-important title="When assigning permissions to roles, ensure they align with the role's intended responsibilities and access requirements."}
 Please note that the `Customer Admin` role has the highest level of access and as such, its permissions cannot be modified.
```
Comment (nrichers) [inline site/guide/configuration/manage-permissions.qmd:21]: Either a missing comma or an extra comma, depending on how comma-tose you are. (I'll see myself out.)

```suggestion
Please note that the `Customer Admin` role has the highest level of access and, as such, its permissions cannot be modified.
```

File: site/guide/configuration/manage-roles.qmd [hunk 2]
```diff
@@ -13,16 +13,32 @@ Managing roles is crucial for maintaining a secure and well-organized environmen
...
+
+1. In the left sidebar, click **{{< fa gear >}} Settings**. 
+
+2. Under User Permissions, select **{{< fa user-group >}} Roles**.
+
+3. Select the role you want to rename.
+
+4. On the role's detail view, click on the title of the role. 
+
+    Type in the new name of the role.
```
Comment (nrichers) [inline site/guide/configuration/manage-roles.qmd:38]: This dangling step looks odd to me. How about this?

```suggestion
5. Type in the new name of the role.
```

**Review by nrichers — APPROVED** [review]
A couple of minor suggested text tweaks but LGTM otherwise. 🚀

**Pull request merged**